files/default/neutron-ha-tool.py

```diff
@@ -21,16 +21,16 @@
 #   2 - router migration required (with --l3-agent-check)
```

Comment:
Would be helpful to include the URL of the ansible playbook repo in the commit message.